### DISCUSSION TITLE
經痛到底有多痛？

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1539] [General]
研究指出，在滿分十分的疼痛量表中
經痛的平均得分，大概介於五到六分
和拔完智齒麻醉消退的痛感差不多

但拔牙之痛，最起碼看得到盡頭
很多人在遇到經痛時，唯一的解法
就是雙手抱胸、膝蓋捲曲、縮成一團
以近乎求饒的姿態消極抵抗這場災難

而且科學家發現，長期經痛
會使大腦中的 #灰質 產生變化：
灰質內傳遞疼痛訊號的區域會減少
但調節疼痛感及內分泌的區域卻會增加

簡單的說，就是盡可能讓痛痛少一點

身為男生沒辦法體會經痛到底有多痛，自己這輩子最痛的感受是被棒球打到🥚🥚，不知道跟經痛有沒有得比勒

**Comment by lyien1021** [comment]
之前有聽過一個形容我覺得蠻貼切的，有人說生理期來時就像有人一直在揍你的肚子一樣。不過我自己覺得最難受的是頭暈、感覺快昏倒，因為這常常導致我沒辦法做任何事情。

**Comment by larry40305** [comment]
經痛是一個常見且困擾許多女性的問題。了解經痛的程度和影響，並尋找適合自己的緩解方法，是重要的。不同人可能對經痛的感受和應對方式有所不同，每個人也需要找到適合自己的方法來減輕痛楚。

**Comment by Miinteaa** [comment]
其實每個人對於疼痛的感受程度都不太一樣，有些人覺得拔牙打麻醉很痛，但我完全沒什麼感覺🤣所以疼痛這件事情好像還是蠻主觀的，雖然量表能夠在一定程度上客觀的紀錄疼痛的程度，但好像還是不太一樣，我自己覺得經痛比較像有人一直捏著你的內臟器官的感覺🥲

**Comment by zhiron544** [comment]
這篇文章讓我對經痛有了更深入的了解，也讓我感受到女性在每個月都要面對的困擾和挑戰。我覺得文章提供了很多有用的資訊和建議，讓我知道如何幫助自己或身邊的女性朋友減輕痛苦。我也認同文章的觀點，每個人對疼痛的感受和應對方式都不一樣，所以不能一概而論或忽視別人的感受。

**Comment by 110302001** [comment]
我有拔過牙的經驗也有經痛的困擾，我覺得這真的蠻看個人的身體狀況，我身邊就有朋友經痛到送急診，我自己沒到那麼誇張，但那種痛感可以想像子宮在被當成一條毛巾在扭著，然後吃止痛藥也沒用，只能說女生真辛苦🥲

**Comment by THH1118** [comment]
網路上有很多女生帶男生去體驗經痛還有生產痛的影片，看起來是真的超痛的，但感覺被棒球打到那邊也不相上下......

**Comment by Sherry9106** [comment]
就我的經驗來看，首先腰會先痠到不行，然後下腹悶痛，大部分的情況都可以忍受，但是會非常不舒服，有時候忍不住了就只能先吃止痛藥，把疼痛感壓下去，除了經期當下的經痛以外，經期前還會伴隨著各種經前症候群⋯⋯

**Comment by yuuu3957** [comment]
經痛還蠻因人而異的，自己之前也跟男性朋友探討過這個問題，因為他真的很好奇經痛的感覺。
那時候給他的回應是，經痛痛苦是在他不是持續性的疼痛，而是一抽一抽的，很像有人規律性的擊打下腹部。同時伴隨著悶的感覺。
我自己是每次都需要吃止痛藥啦，原本以為自己已經很慘，但之前高中同學有痛到只能躺著完全爬不起來，其實有點被嚇到。

**Comment by yurukung** [comment]
其實每個女性的經痛都不太一樣，有那種月經來幾乎沒有感覺的，也有痛到感覺像是被球棒打到，或者是完全站不起來的都有。舉個例子，我身邊有一位女性是從小就認識，那個時候看到她有時候月經來是痛到趴在桌子哭的，後來有繼續連絡才知道，她長大後還有這個症狀，所以去看醫師，醫生診斷他的某些器官可能跟大眾比較不同(我沒有記的精確的名稱)，所以才會這麼痛。
除了經痛，大家知道經期就是女性子宮內膜脫落流血，不斷的出血是會讓人失去體力，昏昏欲睡的。在經期中的女性也是，我自己狀況是不論有沒有經痛，在月經的期間，體力會比一般的情況差，沒辦法出遠門，很容易疲倦想睡。

**Comment by edawdewv** [comment]
我認為經痛的疼痛程度可以媲美牙齒神經痛，甚至有過之而無不及。最大的差別是，牙痛可以馬上看醫生吃藥，但經痛往往只能硬撐。疼痛通常會持續一到兩天，不僅影響情緒，還會讓人做什麼都不順，體質差的人甚至會痛到無法下床，只能蜷縮著度過。最可怕的是，這樣的折磨每個月都會重來一次，讓人無奈又無助。

**Comment by Welly940504** [comment]
對某些人來說，經痛可能只是肚子有點悶悶的、像拉肚子前那種感覺，可以正常上課、上班；但對有些人來說，經痛是會痛到冒冷汗、臉色發白、沒辦法站起來、甚至想吐、想暈倒的那種等級。那不是在誇張，是身體真的在用力「抗議」，像是子宮被拿去擰乾一樣在收縮。

我有聽朋友形容過：像是「肚子裡有一隻小怪獸在用指甲刮子宮」，或是「有人用拳頭一下一下地打你下腹部」，還有人說比骨折還痛，痛到只想縮成一團睡掉一整天。有些人甚至每個月都要吃止痛藥、拿熱水袋敷肚子，才能勉強撐過那幾天。
